### DISCUSSION TITLE
How to get the confidence interval of measured values？

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#1038] [Questions & Problems]
hello team,
It confused me a lot ,II hope I can get your answer,thanks very much!
![image](https://user-images.githubusercontent.com/92715703/144160605-d3627840-53c3-4cf9-b3d1-5cfacbd4e3ae.png)

best,
Carrie.

**Comment by tobiasK2001** [comment]
Dear Carrie,
what exactly was here confusing for you?
Best, Tobias
